README.md

```diff
@@ -4,7 +4,9 @@
 
 ![](images/logos/gordian-signer-screen.jpg)
 
-**Gordian Cosigner** allows users to participate in a multisig by adding a signature to an otherwise unsigned or partially signed PSBT, where they may or may not have initiated the transaction. It can also be used for signing a single-signature PSBT. It's largely intended as an offline signing tool, which allows signing without the usage of a full node: a real wallet is need to both initiate an account map and to initiate a transaction.
+**Gordian Cosigner** allows users to participate in a multisig by adding a signature to an otherwise unsigned or partially signed PSBT that was created on another device. It's largely intended as an offline signing tool, which allows signing without a direct connection to a full node: a real wallet is need to initiate a transaction.
```

Comment:
Can we use some other term than "real wallet"? Wallets are apps, and this is an app, and thus a wallet. What is needed is either wallet that can initiate a PSBT transaction, or transaction coordinator service.

Reply:
Yep. Hadn't even meant to PR this yet.

Reply:
(Done.)